window.py

```diff
@@ -9,9 +9,9 @@
 
 LOGFILE = "sezam.log"
 
-def pathSet(value: str) -> bool:
-    if value == "":
-        print("Attempted to perform an operation without a directory selected!")
+def assert_path(path: str, message: str = "Attempted to perform an operation without a directory selected!") -> bool:
```

Comment:
Btw. you can pass an fstring to this function, e.g. `f"My important error to my important path {path}"`